uaclient/entitlements/livepatch.py

```diff
@@ -240,7 +240,7 @@ def setup_livepatch_config(
                         break
                 if msg == "Unable to enable Livepatch: ":
                     msg += str(e)
-                print(msg)
+                print(util.redact_sensitive_logs(msg))
```

Comment:
Should we redact this when printing the message too ?
Maybe it is a little unnecessary. What do you think @orndorffgrant

Reply:
Ah I was the one who recommended this. But I think you're right @lucasmoura  we really only need the redaction in the subp logging added below.
Sorry for the churn here @CalvoM - my fault - but we can undo this line change.

Reply:
Alright, on it.